### ISSUE TITLE
rollbar:test rake tasks failed in Rails 6.0.0

### ISSUE THREAD

**Issue by Menghongli** [rollbar/rollbar-gem#976]
Running rake rollbar:test fails in Rails 6.0.0
```
Testing manual report...
[Rollbar] Scheduling item
[Rollbar] Sending item
[Rollbar] Sending json
[Rollbar] Success
[Rollbar] Details: https://rollbar.com/instance/uuid?uuid=a6fbacbb-9997-4b52-a85a-c7a53651fcce (only available if report was successful)
Setting up the test app.
Processing...
Test failed! You may have a configuration issue, or you could be using a gem that's blocking the test. Contact support@rollbar.com if you need help troubleshooting.
```

```
env = Rack::MockRequest.env_for("#{protocol}://www.example.com/verify", 'REMOTE_ADDR' => '127.0.0.1')
status, = app.call(env)

puts error_message unless status.to_i == 500
```

Rollbar is expecting a 500 but got 403 instead. which seems to be caused by this new added middleware `ActionDispatch::HostAuthorization`, (https://github.com/rails/rails/pull/33145)

My current workaround is to add `HTTP_HOST` in the header:
```
env = Rack::MockRequest.env_for("#{protocol}://www.example.com/verify", 'REMOTE_ADDR' => '127.0.0.1')
env['HTTP_HOST'] = 'localhost'
status, = app.call(env)

puts error_message unless status.to_i == 500
```

**Comment by waltjones** [comment]
Related: https://github.com/rollbar/rollbar-gem/issues/964

**Comment by Menghongli** [comment]
@waltjones  Thanks for addressing this issue so quickly, but my workaround is just a quick hack, a proper fix is to:

1. Add `www.example.com` as the `HTTP_HOST` in header
2. Add `www.example.com` to the whitelisted host array, but due to the issue in https://github.com/rollbar/rollbar-gem/issues/964, this currently doesn't work.

**Comment by waltjones** [comment]
@Menghongli Thank you, and thanks for your work on this.

If you're also seeing #964, I just posted a follow up question there.

**Comment by Menghongli** [comment]
@waltjones No worries at all, I've post my attempt in #964

**Comment by waltjones** [comment]
Fix released in v2.26.1 https://github.com/rollbar/rollbar-gem/releases/tag/v2.26.1
